main/display/display.h

```diff
@@ -49,7 +51,9 @@ typedef struct
     uint16_t w;
     uint16_t h;
     paletteColor_t * pxFb; // may be null
-} display_t;
+};
+
+typedef struct display display_t;
```

Comment:
Any particular reason for this shuffle?

Reply:
Yes, it's the only way I could get the definitions to work.